src/vm/tests/costs.rs

```diff
@@ -79,6 +79,10 @@ pub fn get_simple_test(function: &NativeFunctions) -> &'static str {
         SetVar => "(var-set var-foo 1)",
         Map => "(map not list-foo)",
         Filter => "(filter not list-foo)",
+        BuffToIntLe => "(buff-to-int-le 0x00000000000000000000000000000001)",
+        BuffToUIntLe => "(buff-to-uint-le 0x00000000000000000000000000000001)",
+        BuffToIntBe => "(buff-to-int-be 0x00000000000000000000000000000001)",
+        BuffToUIntBe => "(buff-to-uint-be 0x00000000000000000000000000000001)",
```

Comment:
Should any more thought be given to this test?

Reply:
This is fine for now -- we'll want more precise testing once the benchmarking work progresses though.